src/Simplex/Chat/Migrations/M20221115_server_cfg.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20221115_server_cfg where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20221115_server_cfg :: Query
+m20221115_server_cfg =
+  [sql|
+PRAGMA ignore_check_constraints=ON;
+
+ALTER TABLE smp_servers ADD COLUMN preset INTEGER DEFAULT 0 CHECK (preset NOT NULL);
+ALTER TABLE smp_servers ADD COLUMN tested INTEGER;
```

Comment:
probably forgot to remove this..
```suggestion
ALTER TABLE smp_servers ADD COLUMN tested INTEGER;
```

Reply:
No, I do think this status need to be saved, I don’t see reason not remember / show servers that were tested as suchC , retesting all when test&save is chosen.